src/vdaf.rs

```diff
@@ -248,6 +248,21 @@ pub trait Aggregator<const VERIFY_KEY_SIZE: usize, const NONCE_SIZE: usize>: Vda
         agg_param: &Self::AggregationParam,
         output_shares: M,
     ) -> Result<Self::AggregateShare, VdafError>;
+
+    #[cfg(feature = "experimental")]
+    /// This is special dpsa-project functionality.
+    /// In order to implement noising for differential privacy,
+    /// We define a custom postprocessing function for vdafs,
+    /// such that each aggregator can call it on its aggregate shares.
+    ///
+    /// The default implementation is the identity function.
+    fn postprocess(
```

Comment:
I agree with Chris that `postprocess` is too vague, and it's too easy to confuse with `prepare_preprocess`. We've already been struggling with the ambiguous "prepare" methods, so I'd like to do better here. I don't see why we wouldn't make this explicitly about DP and just call it `fn apply_differential_privacy_noise`. If we ever need to add more post-processing steps, we'd define new methods on VDAF as well as the order in which they should be called.